src/utils/convertBytes.ts

```diff
@@ -0,0 +1,31 @@
+const bytesPerKB = 1024;
+
+const conversionRates = {
+	KB: 1,
+	MB: 2,
+	GB: 3
+};
+
+export type ByteTypes = 'KB' | 'MB' | 'GB';
+
+/**
+ * Converts bytes to AND from KB, MB, and GB.
+ * By default, it will convert byteCount into the specified unit.
+ *
+ * @param toBytes set to true to convert from KB/MB/GB into bytes
+ * @remarks Decimal count is not considered in this function
+ */
+export default function convertBytes(byteCount: number, unit: ByteTypes, toBytes = false): number {
```

Comment:
How about simplifying and making the API a little easier to understand?
`
const conversionRates = {
	B: 0,
	KB: 1,
	MB: 2,
	GB: 3
};

function convertUnits(count: number, inputUnit: ByteTypes, outputUnit: ByteTypes): number {
	const inputPower = conversionRates[inputUnit] * 10;
	const outputPower = conversionRates[outputUnit] * 10;
	const ret = count / Math.pow(2, outputPower - inputPower);
	return ret;
}
`